src/init.cpp

```diff
@@ -1666,7 +1666,10 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     connOptions.nSendBufferMaxSize = 1000*gArgs.GetArg("-maxsendbuffer", DEFAULT_MAXSENDBUFFER);
     connOptions.nReceiveFloodSize = 1000*gArgs.GetArg("-maxreceivebuffer", DEFAULT_MAXRECEIVEBUFFER);
     connOptions.m_added_nodes = gArgs.GetArgs("-addnode");
-
+    connOptions.m_bantime = gArgs.GetArg("-bantime", DEFAULT_MISBEHAVING_BANTIME);
+    if (connOptions.m_bantime <= 0) {
+        return InitError(strprintf(_("%s cannot be configured with a negative value."), "-bantime"));
```

Comment:
Add test?

Reply:
Could you point me to where bitcoind/cli args are tested? Can't seem to find any example.

Reply:
Search for `assert_raises_process_error`.

Reply:
It is only used in `TestBitcoinCli`, which doesn't look like the right place. It looks like bitcoind command line args are not tested at all.

Shall we keep the check, but untested, like all the others?

Reply:
No, it's not the right place. I was suggesting to add a right place :) IMO they should be tested but I may be alone here.

Reply:
I don't disagree, but I think it's out of scope for this PR. Adding the test suite for this is needs to be prepared separately.

Reply:
You need `assert_start_raises_init_error()`. See https://github.com/bitcoin/bitcoin/blob/90926db2381d87c68858659873230a3811ebdce5/test/functional/multiwallet.py#L26 for example

Reply:
That's right! Sorry for the misleading `assert_raises_process_error`.